src/sections/Projects/Sistent/about.js

```diff
@@ -42,27 +40,47 @@ const SistentAbout = () => {
               <h2>About Sistent</h2>
             </a>
             <p>
-            Sistent is the Layer5 Design System - an open source design system that offers building blocks to create consistent, accessible, and user-friendly interfaces. It's aimed at developers who want to design applications aligned with the same brand and ensure a uniform user experience across different products.
+              Sistent is the Layer5 Design System - an open source design system
+              that offers building blocks to create consistent, accessible, and
+              user-friendly interfaces. It's aimed at developers who want to
+              design applications aligned with the same brand and ensure a
+              uniform user experience across different products.
             </p>
             <p>
-            Sistent leverages Material UI libraries and provides a custom theme on top of it for a consistent look and feel. It includes components, icons, and design tokens that developers can readily integrate into their applications. By using Sistent, developers can save time and effort while maintaining a high-quality user experience throughout Layer5 products.
+              Sistent leverages Material UI libraries and provides a custom
+              theme on top of it for a consistent look and feel. It includes
+              components, icons, and design tokens that developers can readily
+              integrate into their applications. By using Sistent, developers
+              can save time and effort while maintaining a high-quality user
+              experience throughout Layer5 products.
             </p>
             <p>
-              <b>Sistent provides a consistent user experience across Layer5 projects and any frontend software projects that choose to use Sistent, too.</b>
+              <b>
```

Comment:
please try to avoid do lint changes, it make difficult to review

Reply:
Sure, will take  care of that